public/mobile-app/src/routes/+page.svelte

```diff
@@ -83,15 +119,11 @@ const subscribePush = async () => {
 
 const updateButtonsStates = async () => {
   const isGranted = await checkNotificationPermission()
-  console.log('inside updateButtonsStates, isGranted:', isGranted)
   isAuthenticatedForNotifications = isGranted
   isRegisteredWithAmi = !isGranted
 
   if (isGranted) {
-    const pushSubscription = await subscribePush()
-    pushSubURL = pushSubscription.endpoint
-    pushSubAuth = pushSubscription.toJSON().keys.auth
-    pushSubP256DH = pushSubscription.toJSON().keys.p256dh
+    pushSubscription = await subscribePush()
```

Comment:
Is this `updateButtonsStates` function still used? I can't see it anywhere, seems like dead code?